tests/integration_test.py

```diff
@@ -32,7 +32,8 @@ def tearDownClass(cls):
 
     @classmethod
     def generate_hash(cls):
-        return md5('%s%s' % (uuid4(), datetime.now())).hexdigest()
+        hash_feed = '{}{}'.format(uuid4(), datetime.now())
+        return md5(hash_feed.encode('ascii')).hexdigest()
```

Comment:
This will not work in python 3:
```
>>> from hashlib import md5
>>> import six
>>> md5(six.u('terefere'))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: Unicode-objects must be encoded before hashing
>>> 
```